src/Schema/ViewHelperFinder.php

```diff
@@ -22,6 +22,10 @@ final class ViewHelperFinder
     private const FILE_SUFFIX = 'ViewHelper.php';
 
     private ViewHelperMetadataFactory $viewHelperMetadataFactory;
+    /**
+     * @var string[] $lastErrors
+     */
+    private array $lastErrors = [];
```

Comment:
maybe it makes sense to collect the whole exception object here?

Reply:
Done.